pkgs/os-specific/linux/kernel/zen-kernels.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, fetchFromGitHub, buildLinux, ... } @ args:
```

Comment:
I'm not sure this is right, args would apply to both kernels now? I think we should adopt a similar pattern to emacs versions.

Reply:
It seems all works
```
nix-repl> linuxKernel.kernels.zenKernels.zen.kernelPatches
[ { ... } { ... } ]
nix-repl> linuxKernel.kernels.zenKernels.lqx.kernelPatches
[ { ... } { ... } ]
```

Reply:
Problem is that the args might need to be different in the future. zen might be a version ahead and not need a patch anymore for example.

Reply:
I don't understand you: how `args` need to be different in future?
With `buildLinux` derivation changed, all kernels derivation in `pkgs/os-specific/linux/kernel` will be changed too.
And what do you mean by patching?
`zen-kernels` not patching by us, we're already downloading patched version

Reply:
I mean the kernelPatches that are applied to all of our kernels. There might be new ones for newer kernel versions or old ones get obviated. Since zen and lqx have different release cadences, the patches might need to be different aswell.

Reply:
Ooh that's neat, didn't ever think of using partial function application with callPackage. That's definitely the way to go.

Reply:
Should I make these changes or keep them for the future if we are faced with the need to make them?

Reply:
I'd make them now. The kernels are somewhat separate from another, so it makes sense to declare them as such. Maybe add a comment to `zenKernels` to explain why it exists.